MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -291,7 +291,7 @@
                 <Condition Binding="{Binding IsEditable, RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" Value="True" />
               </MultiDataTrigger.Conditions>
               <Setter TargetName="ToggleTemplateRoot" Property="BorderBrush" Value="Transparent" />
-              <Setter TargetName="splitBorder" Property="BorderBrush" Value="{DynamicResource MaterialDesignCheckBoxDisabled}" />
+              <Setter TargetName="splitBorder" Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.CheckBox.Disabled}" />
```

Comment:
Agreed, but this is changing the color of an internal template member to use the disabled brush when the control is disabled

Reply:
Isn't it still a bit odd naming-wise. I mean if a consumer decides to change the `MaterialDesign.Brush.CheckBox.Disabled` brush because he/she wants to affect ONLY the `CheckBox` control, this now also affects the arrow in the `ComboBox`. I think the original intention was to provide consumers with more fine-grained control of the brushes; this seems like an odd side-effect IMO.